pygitguardian/sca_models.py

```diff
@@ -14,11 +14,13 @@ class SCAIgnoredVulnerability(Base, FromDictMixin):
     A model of an ignored vulnerability for SCA. This allows to ignore all occurrences
     of a given vulnerability in a given dependency file.
     - identifier: identifier (currently: GHSA id) of the vulnerability to ignore
+    - cve_id: CVE id of the vulnerability to ignore
     - path: the path to the file in which ignore the vulnerability
     """
 
-    identifier: str
     path: str
+    identifier: Optional[str] = None
+    cve_id: Optional[str] = None
```

Comment:
Making `identifier` optional can be considered to be a breaking change: code using it assuming it's always a string may break if it can now be None.

What do we expect from these fields? Can they both be None, or can they can they both be not None?